idl/chromadb/proto/logservice.proto

```diff
@@ -3,6 +3,17 @@ syntax = "proto3";
 package chroma;
 option go_package = "github.com/chroma/chroma-coordinator/internal/proto/logservicepb";
 
-service LogService {
+import "chromadb/proto/chroma.proto";
+
+message PushLogsRequest {
+  string collection_id = 1;
+  repeated SubmitEmbeddingRecord records = 2;
+}
 
+message PushLogsResponse {
+  int32 record_count = 1;
```

Comment:
Should we also include a status code? Also, can you double check how GRPC handles golang errors in its response to client?

Reply:
I'm confused... There is a Status in the grpc message itself why do we need to have a Status in the response? shouldn't it be the responsibility for the interceptor

Reply:
I think it would be nice to have our own status on different errors for our service. For example, it is useful in SysDB to pass back to the front end why creating a collection is not successful so that the user can take the right action (e.g. unique constraint violation). 

I am not sure how the python side handles the errors raised by the golang GRPC response.

Reply:
My 2 cents, I think we could have both. I understand the value of having specific error code, but also It seems weird to have it in the response. I think we could use grpc response metadata to propagate the custom error code. Keeping native grpc code would help with monitoring and standard tooling. Putting it in metadata, allow us to set it at the framework lvl.